yax/scalatest/src/main/scala/doobie/scalatest/imports.scala

```diff
@@ -1,5 +1,11 @@
 package doobie.scalatest
 
 object imports {
-  type QueryChecker = doobie.scalatest.QueryChecker
+
+  type Checker[M[_]] = doobie.scalatest.Checker[M]
+  type IOLiteChecker = doobie.scalatest.IOLiteChecker
+
+  @deprecated("Use CheckerIOLite.", "0.4.2")
```

Comment:
CheckerIOLite here but below is IOLiteChecker?

Is that an error or am I misunderstanding how this should be used?